client/src/components/NavBar.jsx

```diff
@@ -1,32 +1,31 @@
 import { BsPersonCircle, BsGeoAlt, BsTrophy } from "react-icons/bs";
-import { NavLink } from "react-router-dom";
+import { Link, NavLink } from "react-router-dom";
 import "../styles/NavBar.css";
 
 function Navbar() {
   return (
     <nav className="navbar">
-      <img
-        className="logo"
-        src="./src/assets/images/logo.webp"
-        alt="logo pixhunter"
-      />
+      <Link to="/">
+        <img
+          src="./src/assets/images/logo.webp"
+          alt="logo pixhunter"
```

Comment:
It would be better to use the alt attribute to explain where you will be redirected.

Reply:
Done